src/main/views/crown-firm-list.njk

```diff
@@ -14,26 +14,16 @@
 {% block content %}
   <div class="parent-box">
     <h2 id='page-heading' class="govuk-heading-l">
-      {{ courtName }}: {{ headerPart1 }}
       {% if bill %}
```

Comment:
I think we can simply use "headerPart1" too for bi-lingual. We don't need the "if" statementl. Better still we can rename "headerPart1" to just "header" or "heading".